Canvas/TechDebt/iCanvas/Router+Routes.m

```diff
@@ -262,17 +261,6 @@ - (void)addRoutes
         @"/groups/:contextID/users" : [self usersBlockForClass:[CKIGroup class]],
         @"/courses/:contextID/users/:userID" : [self usersDetailBlockForClass:[CKICourse class]],
         @"/groups/:contextID/users/:userID" : [self usersDetailBlockForClass:[CKIGroup class]],
-        @"/courses/:courseID/quizzes": ^(NSDictionary *params, CBIQuizzesTabViewModel *quizzesTabViewModel) {
```

Comment:
Would it be helpful to put this behind a feature flag?  I feel like i made a mistake doing syllabus w/ out a feature flag b/c it caused problems and some unneeded worry and rushing due to some unforeseen issues.

Reply:
I asked Peyton & David about using a flag, neither seemed to think it was necessary. The list this is replacing was MLVC, so pretty bare bones.